build/js/flow.js

```diff
@@ -8525,6 +8527,7 @@
                 max_after_balance_size: maxAfterBalanceSize,
                 keep_cross_validation_predictions: _keepCrossValidationPredictions(),
                 keep_cross_validation_models: _keepCrossValidationModels(),
+                keepCrossValidationFoldAssignment: _keepCrossValidationFoldAssignment(),
```

Comment:
@michal-raska I just noticed that there might be a naming convention. Should I rename property from camel case to underscore style?